datasync/chaindatafetcher/types.go

```diff
@@ -40,17 +39,17 @@ const (
 // request contains a raw block which should be handled and the type of data which should be exported.
 type request struct {
 	reqType requestType
-	event   blockchain.ChainEvent
+	block   uint64
```

Comment:
```suggestion
	blkNum   uint64
```

Reply:
Renamed to `blockNumber` for consistency.